### PR TITLE
Arrays

### PR DESCRIPTION
**Pull request by ZoyaNaumenko**

### CODE REVIEW

File: Exercises/arrays/arrays.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+// 1. Make a function which will generate array with random length and random values.
+function arrayGenerator(max) {
+  const arraylength = Math.floor(Math.random() * Math.floor(max));
+  const arrayNew = Array.from({ length:arraylength }, () => Math.floor(Math.random() * max));
```
Comment (lizarusi) [inline Exercises/arrays/arrays.js:4]: return new generated array

File: Exercises/arrays/arrays.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  console.log(arraylength, arrayNew);
+}
+arrayGenerator(20);
+
+// 2. Make a function which will take array and return copy of it.
+
+function arrayDuplication(originArray) {
+  const copyArray = originArray.slice();
```
Comment (lizarusi) [inline Exercises/arrays/arrays.js:12]: return new value. Also consider simplier way

File: Exercises/arrays/arrays.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  console.log(copyArray);
+}
+arrayDuplication([1, '45', 6]);
+
+// 3. Make a function which will take array and element and return a index of element in array.
+
+function indexEl(array, element) {
+  const indexElement = array.indexOf(element);
```
Comment (lizarusi) [inline Exercises/arrays/arrays.js:20]: return the index from function

File: Exercises/arrays/arrays.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  console.log(indexElement);
+}
+indexEl([1, '45', 'zoya'], 'zoya');
+
+// 4. Make function which will take two arrays and return association of them.
+function arrayMerge(array1, array2) {
+  const newArray = array1.concat(array2);
```
Comment (lizarusi) [inline Exercises/arrays/arrays.js:27]: consider other way to do it :) and again, return new value from array

File: Exercises/arrays/arrays.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  /*eslint-disable */
+  array1[index] = value;
+  console.log(array1);
+}
+newArray(array1, 0, '100');
+
+const array2 = ['first', 'second', 'third'];
+
+function newArrayPure(array2, index, value) {
+  const copyArray2 = array2;
```
Comment (lizarusi) [inline Exercises/arrays/arrays.js]: it will not be pure, array2 will change

**Review by lizarusi — CHANGES_REQUESTED** [review]

**Review by Mike-Savin — APPROVED** [review]
Looks good

**Pull request merged**